src/utils/setLocalStorage.js

```diff
@@ -0,0 +1,3 @@
+export const setLocalStorage = (questionNum, answer) => {
```

Comment:
I think you can delete this file because Erhan already creating it in this [PR](https://github.com/NouranMahmoud/browser-quiz-project-starter/pull/17/files#r1428094847)